### ISSUE TITLE
NFIMM patch not detected

### ISSUE THREAD

**Issue by shadow2560** [ITotalJustice/sys-patch#1]
When I look at the log of the module I see that NFIM patch is applied by the module but it should be via file because I have the patches made by IPS_Patch_Creator (FW 16.0.3, last version of AMS). Is this behaviour is normal?

Here is the log of the app:
```
[fs]
noacidsigchk1=Skipped
noacidsigchk2=Skipped
noncasigchk_old=Skipped
noncasigchk_new=Patched (file)
nocntchk_old=Skipped
nocntchk_new=Patched (file)
[ldr]
noacidsigchk=Patched (file)
[es]
es1=Skipped
es2=Skipped
es3=Skipped
es4=Skipped
es5=Skipped
es6=Patched (file)
[nifm]
ctest=Patched (sys-patch)
[stats]
version=1.4.2-e813595
build_date=28.05.2023 01:21:37
fw_version=16.0.3
ams_version=1.5.4
ams_target_version=0.3.0
ams_keygen=15
ams_hash=3cb54e2b
is_emummc=1
heap_size=4096
buffer_size=4096
patch_time=0.079s

```

**Comment by ITotalJustice** [comment]
Hi. I think this is because sigpatches patch nifm differently to me. My patch adds a sleep, which should fix issues some people had with the nifm patch where it would (extremely rarely) have core 3 run at 100%.

This is my patch https://armconverter.com/?disasm&lock=arm64&code=00309AD2001EA1F2610100D4E0031FAAC0035FD6

This is what sigpatches patch. https://armconverter.com/?disasm&lock=arm64&code=E0031FAAC0035FD6

Basically, sys-patch is overwriting what sigpatches set. This is perfectly fine.

**Comment by ITotalJustice** [comment]
I say "I think sigpatches patch differently to me" because I'm not sure if sigpatches are updated with the patch I sent. We couldn't reproduce the core3 bug so it was difficult to say for definite that my patch would actually fix it, so maybe they ignored it for now :)

**Comment by shadow2560** [comment]
Thanks a lot for the explanations.

For me no bug to report with the two patches so it was justt a thing that I wanted you to know if this was an error or a bug (I've never understand how realy sig_patches work, I tried but it seems to be out of my skills), hapy to learn that it wasn't.

And big thanks for your excellent work.
